myRental/modules/database.js

```diff
@@ -10,24 +10,27 @@ let Log = require('./Log.js');
 
 let logger = new Log();
 let User = null;
+let Request = null;
 
 function dataBaseModule(type) {
```

Comment:
In Laravel, remember how there were facades that did that for us. Maybe we could extract this code to a file that we could use like that. It imports both sets of models, checks the type, then returns those models.

Reply:
Possibly, I will look into this.